tests/unit/engine/test_setup_transform.py

```diff
@@ -41,17 +41,19 @@ def __init__(self) -> None:
         super().__init__()
         self.model = torch.nn.Linear(10, 10)
 
-    def configure_transforms(self, image_size: tuple[int, int] | None = None) -> Transform:
+    def configure_transforms(self, image_size: tuple[int, int] | None = None) -> Transform:  # noqa: PLR6301
```

Comment:
Why can't we make this a static method?

Reply:
I was not sure if any of the models would need `self` in the transforms. If you think it is safe to make it static method, I could change that

Reply:
I double checked, and converted the child class implementations. I left the some of the parent classes as is though.